### PR TITLE
Changes to cf to make it pipeline ready

### PR DESCRIPTION
**Pull request by jonhartzler**
PR to have a conversation about what I am doing. 

So Far
1.  Changed the templates to SAM to make it easier to deploy in the pipeline. '
2.  Added in scripts that the blueprints pipeline can reference in the build spec.  
3.  Remove the cloudformation from this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,3 +135,10 @@ To run using the example file, navigate to directory /manifest-from-input-json a
 ```python3 create_manifest.py ../example/ example-input.json ./```
 
 A JSON manifest file will be created in the output directory, and the JSON output will also be printed.
+
+
+## aws deployement temp
+
+aws cloudformation deploy --template-file /Users/jhartzle/Workspace/mellon-manifest-pipeline/output.yml --stack-name manifest-pipeline-dev1
```
Comment (ialford) [inline README.md]: We'll probably want to just say 
```console
cd mellon-manifest-pipeline
aws cloudformation deploy --template-file output.yml --stack-name name-of-stack

File: deploy/pipeline.yml [hunk 2]
```diff
@@ -155,6 +155,7 @@ Resources:
               - 's3:GetObject'
               - 's3:GetObjectVersion'
               - 's3:GetBucketVersioning'
+              - 'ssm:*'
```
Comment (ialford) [inline deploy/pipeline.yml]: I would suggest you scope this a bit more: are you just needing the ability to read parameters, and if so just on a certain path? You can take a look at https://github.com/ndlib/remix-blueprints/blob/master/deploy/cloudformation/static-host-pipeline.yml#L117-L141 for an example of allowing a full path to be read.

File: prod-stack-configuration.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "Parameters" : {
+    "EnvType" : "production"
+  },
+  "Tags" : {
+    "Name" : "testlibnd-appstream-gateway",
+    "Contact" : "lib-esu-group@nd.edu",
+    "Owner" : "wsill",
```
Comment (jgondron) [inline prod-stack-configuration.json]: I support you in blaming Bill for everything that might happen with this pipeline 💯

Comment (jonhartzler) [reply prod-stack-configuration.json]: i am !!

**Pull request merged**